src/Beberlei/Metrics/Collector/Logger.php

```diff
@@ -15,36 +15,62 @@
 
 use Psr\Log\LoggerInterface;
 
-class Logger implements Collector
+class Logger implements GaugeableCollector
 {
+    /** @var \Psr\Log\LoggerInterface */
     private $logger;
 
+    /**
+     * @param \Psr\Log\LoggerInterface $logger
+     */
     public function __construct(LoggerInterface $logger)
     {
         $this->logger = $logger;
     }
 
+    /**
+     * {@inheritDoc}
+     */
     public function increment($variable)
     {
-        $this->logger->debug("increment:$variable");
+        $this->logger->debug('increment:' . $variable);
     }
 
+    /**
+     * {@inheritDoc}
+     */
     public function decrement($variable)
     {
-        $this->logger->debug("decrement:$variable");
+        $this->logger->debug('decrement:' . $variable);
     }
 
+    /**
+     * {@inheritDoc}
+     */
     public function timing($variable, $time)
     {
-        $this->logger->debug("timing:$variable:$time");
+        $this->logger->debug('timing:' . $variable . ':' . $time);
```

Comment:
Good point @lyrixx, I'll use `sprintf()` here as well.

@stof there's nowhere in the project that says it uses the Symfony rule. The most important is that it stays as much consistent as possible. Maybe it worths adding it in the a `Contribution.md` file.